2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_setup.py:_flush():71] setting env: {}
2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_155436-1woflnrr/logs/debug.log
2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_155436-1woflnrr/logs/debug-internal.log
2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_init.py:init():404] calling init triggers
2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-28 15:54:36,275 INFO    MainThread:626061 [wandb_init.py:init():460] starting backend
2022-02-28 15:54:36,275 INFO    MainThread:626061 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-28 15:54:36,281 INFO    MainThread:626061 [backend.py:ensure_launched():216] starting backend process...
2022-02-28 15:54:36,285 INFO    MainThread:626061 [backend.py:ensure_launched():222] started backend process with pid: 626070
2022-02-28 15:54:36,286 INFO    MainThread:626061 [wandb_init.py:init():469] backend started and connected
2022-02-28 15:54:36,287 INFO    MainThread:626061 [wandb_init.py:init():533] updated telemetry
2022-02-28 15:54:36,288 INFO    MainThread:626061 [wandb_init.py:init():563] communicating current version
2022-02-28 15:54:36,738 INFO    MainThread:626061 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-28 15:54:36,738 INFO    MainThread:626061 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-28 15:54:37,241 INFO    MainThread:626061 [wandb_init.py:init():606] starting run threads in backend
2022-02-28 15:54:39,846 INFO    MainThread:626061 [wandb_run.py:_console_start():1810] atexit reg
2022-02-28 15:54:39,847 INFO    MainThread:626061 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-28 15:54:39,847 INFO    MainThread:626061 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-28 15:54:39,848 INFO    MainThread:626061 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-28 15:54:39,848 INFO    MainThread:626061 [wandb_init.py:init():633] run started, returning control to user process
2022-02-28 15:54:41,480 INFO    MainThread:626061 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-28 15:54:41,481 INFO    MainThread:626061 [wandb_run.py:_restore():1752] restore
2022-02-28 15:54:43,858 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 15:54:44,273 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 15:54:45,251 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 6090
}

2022-02-28 15:54:45,353 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15715
}

2022-02-28 15:54:45,455 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15715
}

2022-02-28 15:54:45,557 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15715
}

2022-02-28 15:54:45,659 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15400
  total_bytes: 15715
}

2022-02-28 15:54:45,761 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15400
  total_bytes: 15715
}

2022-02-28 15:54:45,863 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15715
  total_bytes: 15715
}

2022-02-28 15:54:45,965 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15715
  total_bytes: 15715
}

2022-02-28 15:54:46,067 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15715
  total_bytes: 15715
}

2022-02-28 15:54:46,169 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15715
  total_bytes: 15715
}

2022-02-28 15:54:46,271 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15715
  total_bytes: 15715
}

2022-02-28 15:54:46,705 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15715
  total_bytes: 15715
}

2022-02-28 15:54:47,356 INFO    MainThread:626061 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15715
  total_bytes: 15715
}
local_info {
}

2022-02-28 15:54:48,421 INFO    MainThread:626061 [wandb_run.py:_append_files():2180] logging synced files
